src/couch_mrview/src/couch_mrview_http.erl

```diff
@@ -465,6 +466,14 @@ parse_params(Props, Keys, #mrargs{}=Args0, Options) ->
         parse_param(K, V, Acc, IsDecoded)
     end, Args1, Props).
 
+
+parse_body_and_query(#httpd{method='POST'} = Req, Keys) ->
+    Props = chttpd:json_body_obj(Req),
+    parse_body_and_query(Req, Props, Keys);
+
+parse_body_and_query(Req, Keys) ->
+    parse_params(chttpd:qs(Req), Keys, #mrargs{keys=Keys, group=undefined, group_level=undefined}, [keep_group_level]).
```

Comment:
Project style is to wrap lines at 80 columns.

Reply:
Fixed in b025b2c36ff6ee4c82689e3fcd612623852c59f0